### PR TITLE
Added node as script headers

### PR DESCRIPTION
**Pull request by rponnuru0518**

### CODE REVIEW

**Review by kailasammoni — APPROVED** [review]

File: reactAsScriptTag.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (rajender463) [inline reactAsScriptTag.html:1]: This file is good, but we do not need to do these things.
npm comand to create react app will be able to do all these things.

Comment (rajender463) [reply reactAsScriptTag.html:1]: `npx create-react-app your-app-name-here --template typescript`.
This will create basic stuff for us.

Comment (ranathummeti) [reply reactAsScriptTag.html:1]: @rajender463 @rponnuru0518  Can you enable build commands for app ? I can work on getting CI ready using github actions or travis ci.  I will set up a CD to deploy to github pages. 

I might require build commands for building the application. 

eg : `npm build` or `npm run build`

Comment (rajender463) [reply reactAsScriptTag.html:1]: @ranathummeti sure, we will be using webpack.
So need little bit of research before configuring that build command

Comment (rajender463) [reply reactAsScriptTag.html:1]: @ranathummeti @rponnuru0518 So we consider of having one environment which referring as Dev??

Comment (rponnuru0518) [reply reactAsScriptTag.html:1]: @rajender463 Yes, I will create another app with npx command tomorrow. As the name of the file describes, this example is for react as a script tag. @ranathummeti  yes, when we have some js files. npm build will be used to build.

Comment (rponnuru0518) [reply reactAsScriptTag.html:1]: About the environment i have to look into github pages, but sure we can have as many for learning purpose.

**Pull request merged**